tokio-executor/src/threadpool/builder.rs

```diff
@@ -374,7 +374,7 @@ impl Builder {
     ///     .build();
     /// ```
     pub fn build(&self) -> ThreadPool {
-        trace!("build; num-workers={}", self.pool_size);
+        trace!(message = "build;", num_workers = self.pool_size);
```

Comment:
Why does this require `message = ` instead of the first argument being the message?